contracts/access/AccessControlDefaultAdminRules.sol

```diff
@@ -207,9 +249,27 @@ abstract contract AccessControlDefaultAdminRules is IAccessControlDefaultAdminRu
      * Internal function without access restriction.
      */
     function _beginDefaultAdminTransfer(address newAdmin) internal virtual {
```

Comment:
Initially we said that an admin transfer wouldn't be allowed if there was a delay change in progress, but I don't see that in this function. Did you remove this on purpose? Can you share your reasoning? Honestly I don't think I ever had a good intuition for why we are/were not allowing these two things to happen in parallel. I see that initiating a delay change is still not allowed if there is an admin transfer in progress.

Reply:
Initially it was because of assumptions we eventually dropped. I didn't want to make this more complex because it was hard to reason what delay was in effect since it required a "materialization".

This last iteration is way more straightforward and well-encapsulated, so I decided to relax the restriction of both changes not occurring in parallel. It's easy to see what is the delay change in every moment.